conf/login_chat.conf.dist

```diff
@@ -6,6 +6,4 @@
 # to make it available in this table.
 #
 
-LoginChat.enabled=true
```

Comment:
maybe keep this option with related code? it's useful

Reply:
this is dont used in code and don't work at all

Reply:
ok xD